Explorer/Assets/DCL/SDKComponents/AvatarModifierArea/Components/AvatarModifierAreaComponent.cs

```diff
@@ -0,0 +1,27 @@
+using Google.Protobuf.Collections;
+using System.Collections.Generic;
+
+namespace DCL.SDKComponents.AvatarModifierArea.Components
+{
+    public struct AvatarModifierAreaComponent
+    {
+        public readonly HashSet<string> ExcludedIds;
+
+        public AvatarModifierAreaComponent(RepeatedField<string> excludedIds)
+        {
+            ExcludedIds = new HashSet<string>();
+            SetExcludedIds(excludedIds);
+        }
+
+        public void SetExcludedIds(RepeatedField<string> excludedIds)
+        {
+            ExcludedIds.Clear();
+
+            foreach (string excludedId in excludedIds)
+            {
+                // Profile component stores UserId in lowercase...
+                ExcludedIds.Add(excludedId.ToLower());
+            }
+        }
+    }
```

Comment:
RepeatedField can be replaced by IEnumerable

Reply:
also with this ToLower transformation that are spreading around the code base let's bring here `struct UserId` that automatically translated it to lower case

Reply:
Tackled in [this commit](https://github.com/decentraland/unity-explorer/pull/336/commits/2ff94b5bfe636a43c73c3a8616c9bca7b55d70cb)